src/reboost/build_glm.py

```diff
@@ -174,8 +176,8 @@ def get_stp_evtids(
 
 
 def build_glm(
-    stp_file: str,
-    glm_file: str | None,
+    stp_files: str,
+    glm_files: str | None,
```

Comment:
```suggestion
    stp_files: str | list[str],
    glm_files: str | list[str] | None,
```

please also for `build_hit`, I cannot add a review comment there...